kv-components/vue/stories/KvMaterialIcon.stories.js

```diff
@@ -0,0 +1,155 @@
+import KvMaterialIcon from '../KvMaterialIcon.vue';
+
+function getFilenames(r) {
+	return r.keys().map((filename) => {
+		let str = filename;
+		str = str.substring(2); // remove ./
+		str = str.substring(0, str.length - 4); // remove .ext
+		return str;
+	});
+}
+
+export default {
+	title: 'KvMaterialIcon',
+	component: KvMaterialIcon,
+	argTypes: {
+		name: {
+			control: {
+				disable: true, // disable the list of all icons since we only use them on one story
+			},
+		},
+	},
+};
+
+const DefaultTemplate = (args, { argTypes }) => ({
+	props: Object.keys(argTypes),
+	components: { KvMaterialIcon },
+	template: `
+		<kv-material-icon
+			:name="name"
+		/>`,
+});
+
+export const Default = DefaultTemplate.bind({});
+Default.args = {
+	name: 'Cart',
+};
+Default.argTypes = {
+	name: {
+		control: {
+			type: 'select',
+			options: getFilenames(require.context('vue-material-design-icons/', true, /\.vue$/)),
```

Comment:
@ryan-ludwig I'm pretty sure this is why you were getting the longer compiles for storybook. Which... may or may not be ok for Storybook. Maybe we should instead only show a subset instead of the full list available from mdi.

This wouldn't be part of the ui app though b/c we'd only dynamically import what need in context.

Reply:
Actually, I removed this and event without intentionally looping through ALL icons they are all still processed leading to the slowed down builds and performance of storybook. So, I think it's either internal to the imported component or a result of using the computed based dynamic import method...